src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -164,15 +164,9 @@ def __init__(
             self.power_distributor_exponent
         )
 
-        batteries = component_graph.components(
-            component_category={ComponentCategory.BATTERY}
-        )
-
-        self._battery_pool = BatteryPoolStatus(
-            battery_ids={battery.component_id for battery in batteries},
-            max_blocking_duration_sec=30.0,
-            max_data_age_sec=10.0,
-        )
+        # This will be temporary initialized in the `run` method.
+        # PowerDistributingActor won't use BatteryPoolStatus soon.
+        self._battery_pool: BatteryPoolStatus
```

Comment:
I see just a single use of this variable outside the `run` method.  I think it might be cleaner to make it a local variable inside the `run` function,  and pass it as argument from there.

Reply:
Done ( in new commit)